CONTRIBUTING.md

```diff
@@ -59,9 +56,30 @@ If your code requires new dependencies, be sure that those are available on all
 
 ### User experience
 
-Jamulus is used by people all over the world with different backgrounds and knowledge. Features and language should be user focused.
-The Jamulus project has a [style and tone guide](https://jamulus.io/contribute/Style-and-Tone) which you should follow to remain consistent.
-Features should be usable in the sense that they act as expected to somebody who does not have a technical background.
+Jamulus is used by people all over the world with different backgrounds and knowledge. Therefore, think user focused.
```

Comment:
This is rather vapid. What are we trying to say, exactly?

Reply:
Something like ensure that the app is usable by people who don't know computers well. 
Sorry for the stereotypical image here:
Ensure the feature is usable by a person born without having grown up with computers (I.e. (great) grand father)

Reply:
That's pretty much what we're saying in the next paragraph with "Features should be usable... somebody who does not have a technical background." 

Maybe just combine them into one para?

"People from all over the world and with different levels of knowledge use Jamulus. Features should be usable in the sense that they act as expected to somebody who does not have a technical background."

Reply:
Ok. Updated this part slightly.